docs/user-guide/deployments-administration/upgrade.md

```diff
@@ -5,26 +5,286 @@ description: Introduce how to upgrade GreptimeDB to the latest version, includin
 
 # Upgrade
 
-## How to Upgrade GreptimeDB
+## Overview
 
-If your currently deployed GreptimeDB version is v0.12 or higher, v0.14 is fully compatible with existing database configurations and data formats, allowing for a direct upgrade. If you are using a version prior to v0.12, it is recommended to first refer to the v0.12 upgrade documentation to upgrade your database to v0.12, and then upgrade to v0.14, to ensure compatibility and a smooth transition.
+This guide provides upgrade instructions for GreptimeDB, including compatibility information and breaking changes for each version. Before upgrading, ensure you review the relevant breaking changes for your upgrade path.
 
+## Upgrade Paths to v1.0
+
+### From v0.16 to v1.0
+
+If you are currently running v0.16, you can upgrade directly to v1.0. Review all breaking changes from v0.17 and v1.0 releases below.
```

Comment:
Better to link the section below such as `[Upgrading from v0.16 to v1.0](#Upgrading-from-v0.16-to -1.0)` like that.